### PR TITLE
Added copy as getter to basic array element

### PR DESCRIPTION
**Pull request by pksunkara**
@smizell @danielgtaylor Can you review this? What do you think about it?

### CODE REVIEW

**Comment by pksunkara** [comment]
@tu1ly copy support in this PR.

File: src/elements/copy.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 export default function(namespace) {
   const StringElement = namespace.getElementClass('string');
+  const ArrayElement = namespace.getElementClass('array');
+
+  ArrayElement.prototype.__defineGetter__('copy', function() {
+    return this.children((item) => item.element === 'copy');
+  });
+
+  namespace.register('array', ArrayElement);
```
Comment (danielgtaylor) [inline src/elements/copy.js]: Is this register necessary? Aren't you modifying the prototype of the already-registered class in-place?

**Comment by danielgtaylor** [comment]
@pksunkara just had the one comment, otherwise I think this looks good. I think modifying the array element makes sense in this case!

**Comment by pksunkara** [comment]
@danielgtaylor Fixed it in #14

**Pull request merged**